can-bus/json/DCM/DCM_tx.json

```diff
@@ -8,5 +8,16 @@
                 "bits": 1
             }
         }
+    },
+    "DCMVitals": {
```

Comment:
Had to add vitals signals to other boards... wondering I should rollback theses changes to avoid inevitable merge conflicts

Reply:
These look good, let's leave them